installer/msi/BinaryFragment.wxs

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2020-Present MongoDB Inc. -->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    <DirectoryRef Id="BIN">
+	    <?if $(var.Platform)=x64 ?>
+	       <?define win64Flag = "yes" ?>
+	    <?else ?>
+	       <?define win64Flag = "no" ?>
+	    <?endif ?>
+       <Component Guid="{02DA3AB6-9773-4B13-9D02-7874FE99050D}" Id="RegKeys" Win64="$(var.win64Flag)">
+           <RegistryKey ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes" Id="MongoDBKey" Key="SOFTWARE\MongoDB\[ProductName]" Root="HKLM" />
+           <RegistryValue Id="Version" Key="SOFTWARE\MongoDB\[ProductName]"  Name="Version" Root="HKLM" Type="string" Value="[ProductVersion]" />
+       </Component>
+       <Component Guid="{C01C3018-0321-458D-925D-73E70FE680D3}" Id="bsondump" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="bsondump.exe"  Name="bsondump.exe" Source="bsondump.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0322-458D-925D-73E70FE680D3}" Id="mongodump" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="mongodump.exe"  Name="mongodump.exe" Source="mongodump.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0323-458D-925D-73E70FE680D3}" Id="mongoexport" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="mongoexport.exe"  Name="mongoexport.exe" Source="mongoexport.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0324-458D-925D-73E70FE680D3}" Id="mongofiles" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="mongofiles.exe"  Name="mongofiles.exe" Source="mongofiles.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0325-458D-925D-73E70FE680D3}" Id="mongoimport" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="mongoimport.exe"  Name="mongoimport.exe" Source="mongoimport.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0327-458D-925D-73E70FE680D3}" Id="mongorestore" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="mongorestore.exe"  Name="mongorestore.exe" Source="mongorestore.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0328-458D-925D-73E70FE680D3}" Id="mongostat" Win64="$(var.win64Flag)">
+            <File DiskId="1" Id="mongostat.exe"  Name="mongostat.exe" Source="mongostat.exe" />
+       </Component>
+       <Component Guid="{C01C3018-0329-458D-925D-73E70FE680D3}" Id="mongotop" Win64="$(var.win64Flag)">
```

Comment:
is just incrementing part of the UUID kosher? I feel like we should actually be using a generator to get new ones.

Reply:
It should be fine, but I don't mind generating new ones.

Reply:
Note: the UUIDs are not used for cryptography, and I'm assuming they are uniformly distributed. But I generated new ones, so it's moot :)